src/cobra/test/test_core/test_core_reaction.py

```diff
@@ -390,72 +409,108 @@ def test_removal_from_model_retains_bounds(model):
     assert reaction._upper_bound == 1000.0
 
 
-def test_set_bounds_scenario_1(model):
+def test_set_bounds_scenario_1(model: Model) -> None:
+    """Test reaction bounds setting for a scenario."""
     acald_reaction = model.reactions.ACALD
     assert acald_reaction.lower_bound == -1000.0
     assert acald_reaction.upper_bound == 1000.0
     assert acald_reaction.forward_variable.lb == 0.0
     assert acald_reaction.forward_variable.ub == 1000.0
     assert acald_reaction.reverse_variable.lb == 0
     assert acald_reaction.reverse_variable.ub == 1000.0
-    acald_reaction.upper_bound = acald_reaction.lower_bound - 100
-    assert acald_reaction.lower_bound == -1100.0
-    assert acald_reaction.upper_bound == -1100.0
-    assert acald_reaction.forward_variable.lb == 0
-    assert acald_reaction.forward_variable.ub == 0
-    assert acald_reaction.reverse_variable.lb == 1100.0
-    assert acald_reaction.reverse_variable.ub == 1100.0
-    acald_reaction.upper_bound = 100
-    assert acald_reaction.lower_bound == -1100.0
-    assert acald_reaction.upper_bound == 100
-    assert acald_reaction.forward_variable.lb == 0
-    assert acald_reaction.forward_variable.ub == 100
+    with pytest.raises(ValueError):
+        acald_reaction.bounds = (
+            acald_reaction.lower_bound,
+            acald_reaction.lower_bound - 100,
+        )
+        assert acald_reaction.lower_bound == -1100.0
+        assert acald_reaction.upper_bound == -1100.0
+        assert acald_reaction.forward_variable.lb == 0
+        assert acald_reaction.forward_variable.ub == 0
+        assert acald_reaction.reverse_variable.lb == 1100.0
+        assert acald_reaction.reverse_variable.ub == 1100.0
+        acald_reaction.bounds = (acald_reaction.lower_bound, 100)
```

Comment:
I would just keep this as it was:

```suggestion
        acald_reaction.upper_bound = 100
```

Reply:
Do we want to keep `upper_bound` and `lower_bound` setters?

Reply:
Yes, the individual setters will remain in effect.

Reply:
Okay I see.